### PR TITLE
doc: Be consistent with use of the "sudo" - v3

### PR DESCRIPTION
**Pull request by 0xEniola**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/5720

Previous PR: https://github.com/OISF/suricata/pull/9568

Describe changes:
- Incorporate feedback from last PR.
- Changed the `Important` directives to `note` directives.

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -90,18 +90,20 @@ Rust support::
 Ubuntu/Debian
 """""""""""""
 
+.. note:: The following instructions require `sudo` be installed.
```
Comment (jufajardini) [inline doc/userguide/install.rst:93]: nit: 
```suggestion
.. note:: The following instructions require `sudo` to be installed.
```

Comment (jufajardini) [reply doc/userguide/install.rst:93]: Same for the other notes ;)

File: doc/userguide/install.rst [hunk 2]
```diff
@@ -193,6 +199,8 @@ After installing you can proceed to the :ref:`Basic setup`.
 Upgrading
 """""""""
 
+.. note:: The following instructions require `sudo` be installed.
+
```
Comment (jufajardini) [inline doc/userguide/install.rst:203]: As removing, upgrading or running Suricata would only be possible if one had previously installed it, I don't think we need the note about installing `sudo` or in the sections that are not about installing Suri.

**Review by jufajardini — CHANGES_REQUESTED** [review]
CI :  running
Code :  doesn't apply
Commits segmentation :  ok
Commit author:  ok
Commit messages : ok 
Ticket number on message : ok
CLA :  (to be confirmed)
Doc update :  see inline comments ;)
Redmine ticket :  ok
Rustfmt :  doesn't apply
Tests : 
Dependencies added:  none

**Comment by jufajardini** [comment]
One other thing I couldn't comment inline: for the `Debian` section, don't the lines after the `echo` command require the usage of `sudo` as well?

File: doc/userguide/install.rst (hunk 2 above)
Comment (0xEniola) [reply doc/userguide/install.rst:203]: So you're suggesting the notes there are not necessary then.

You're right, still. 

OK.
Working on it now.

Comment (jufajardini) [reply doc/userguide/install.rst:203]: 😺

**Pull request closed without merging**